website/content/docs/services/discovery/dns-overview.mdx

```diff
@@ -0,0 +1,41 @@
+---
+layout: docs
+page_title: DNS Usage Overview 
+description: >-
+  For service discovery use cases, Domain Name Service (DNS) is the main interface to look up, query, and address Consul nodes and services. Learn how a Consul DNS lookup can help you find services by tag, name, namespace, partition, datacenter, or domain.
+---
+
+# DNS Usage Overview
+
+This topic provides overview information about how to look up Consul nodes and services using the Consul DNS. 
+
+## Consul DNS
+The primary interface for discovering services registered in the Consul catalog is the Consul DNS. The DNS enables you to look up services and nodes registered with Consul using terminal commands instead of making HTTP API requests to Consul. 
```

Comment:
Active voice 📣 
```suggestion
The Consul DNS is the primary interface for discovering services registered in the Consul catalog. The DNS enables you to look up services and nodes registered with Consul using terminal commands instead of making HTTP API requests to Consul. 
```